crates/cli/src/cfg.rs

```diff
@@ -126,16 +135,38 @@ pub enum DALayerOption {
     InMemory,
 }
 
+#[derive(Debug, Default, Clone, Eq, PartialEq, Serialize, Deserialize, ValueEnum)]
+pub enum DBValues {
+    #[default]
+    RocksDB,
+    InMemory,
+    Redis,
+}
+
+#[derive(Args, Deserialize, Clone, Debug)]
+pub struct DatabaseArgs {
+    #[arg(long, value_enum)]
+    db_type: Option<DBValues>,
+
+    /// Path to the RocksDB database, used when `db_type` is `rocks-db`
+    #[arg(long)]
+    rocksdb_path: Option<String>,
+
+    /// Connection string to Redis, used when `db_type` is `redis`
+    #[arg(long)]
+    redis_url: Option<String>,
```

Comment:
Improve error handling:
If `db_type` is `redis` throw an error saying the user needs to set `redis_url`.

Reply:
```
❯ ./target/release/prism-cli prover -s 500 --network-name="custom4" --db-type redis
error: the following required arguments were not provided:
  --redis-url <REDIS_URL>

Usage: prism-cli prover --celestia-start-height <CELESTIA_START_HEIGHT> --network-name <NETWORK_NAME> --db-type <DB_TYPE> --redis-url <REDIS_URL>

For more information, try '--help'.
```